src/main/java/com/yy5/employee/service/EmployeeService.java

```diff
@@ -31,5 +30,29 @@ public Employee insert (String name, int age ) {
         employeeMapper.insert(employee);
         return employee;
     }
+
+    public void update(int employeeNumber, String name, Integer age) throws EmployeeNotFoundException {
+        Employee employee = findEmployee(employeeNumber);
+        if (employee == null) {
+            throw new EmployeeNotFoundException("employeeNumberが" + employeeNumber + "の社員は存在しません");
+        }
+        validateUpdateParameters(name, age);
+        performUpdate(employeeNumber, name, age);
+    }
+
+    private void performUpdate(int employeeNumber, String name, Integer age) {
+        employeeMapper.update(employeeNumber, name, age);
+    }
```

Comment:
わざわざ、performUpdateという関数にしないで、employeeMapper.updateをそのまま使ってしまって良いですよ！
小さく関数を切りすぎると、関数を行ったり来たりしないといけなくなるので、逆にソースコードの見通しが悪くなります！
```suggestion
        employeeMapper.update(employeeNumber, name, age);
    }
```

Reply:
performUpdateメソッドを削除し、updateメソッド内で完結するように修正しました。
[8e516f6](https://github.com/YYprogramer/employee/pull/14/commits/8e516f6eb257591afe5cffc45348a618d3cb22da)